circus-train-integration-tests/src/test/data/com/hotels/bdp/circustrain/integration/CircusTrainHdfsHdfsIntegrationTest/partitioned-single-table-no-housekeeping-with-table-parameters.yml

```diff
@@ -0,0 +1,19 @@
+table-replications:
+  - source-table:
+      database-name: ${circus-train-runner.database-name}
+      table-name: ct_table_p
+      partition-filter: (continent='Europe' AND country='UK') OR (continent='Asia' AND country='China')
+      partition-limit: 100
+    replica-table:
+      table-location: ${circus-train-runner.replica-warehouse-uri}/${circus-train-runner.database-name}/ct_table_p
+    transform-options:
+      table-parameters:
+        '[table.parameter.first]': first
```

Comment:
Is the need for the `'[` due to yml (or something else) to prevent it from doing unwanted things with those `.`s?

Reply:
They are, without the '[]' spring will read the property as a nested map, and not as a single String.